editors/code/package.json

```diff
@@ -653,17 +740,17 @@
                 "title": "$generated-start"
             },
             {
-                "title": "assist",
+                "title": "Assist",
```

Comment:
Below this point, all the changes are generated from config.rs. There are no manual changes to this file below here. The changes are generally:
- category name is title case
- description is imperative